app/controllers/welcome_controller.rb

```diff
@@ -1,4 +1,11 @@
 class WelcomeController < ApplicationController
   def index
+    results = FoodService.search_for_foods(params[:q])
```

Comment:
It seems like you started to pull this logic into a facade but maybe you didn't get to fully refactoring.  Generally t his is exactly what the facade can handle.

Reply:
Keep in mind as well that we do want to keep the original form on the welcome page, so invoking the food facade would make more sense from a foods controller. With this solution, we don't have a chance to enter query params or fill out the form at all